mantidproject/mantid
# Plot Help docs

**Description of work.**

Adding plot help documentation, accessible via help button (added in #28750 )

**To test:**



Build mantid and docs, 
produce one of each: 1D Plot, Tiled Plot, Colorfill, Contour, 3D Wireframe and 3D Surface Plots
Click the help toolbar button and check the correct documentation is linked 

AND...
That it is useful, correct for all toolbar buttons, options menus, click menus, example code and links work!

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
